lib/db_tests.js

```diff
@@ -82,6 +106,8 @@ describe('Database API', function() {
   });
 
   it('should create multiple entities', function(done) {
+    user2.a = 2;
```

Comment:
Why is this added now, were they missing? Where is `user1.a`?

Reply:
those records are created in previous steps, user1 is created, modified and then deleted

Reply:
OK I see them now initialised on the top, my bad!